appveyor.yml

```diff
@@ -14,3 +14,9 @@ build:
 configuration:
 - Debug
 - Release
+test:
+  assemblies:
```

Comment:
I didn't figure out why HttpMock 1.3.3 will get `Has no TestFixtures`, but the old version are fine. And it seems there are a few  `Has no TestFixtures` issues in nunit3. Just open a new issue about this one. 

But still, is HttpMock.dll supposed to be a test assembly to run the tests, in fact there is no tests in it. 
>Tests run: 0, Errors: 0, Failures: 0, Inconclusive: 0, Time: 0.00674923043719606 seconds
  Not run: 0, Invalid: 0, Ignored: 0, Skipped: 0
And I assume we don't care if we run HttpMock.dll, right?

And unfortunately, it is impossible  to exclude a specific assembly in AppVeyor...